provisioning/miniprov/hare_mp/main.py

```diff
@@ -254,8 +275,9 @@ def prepare(args):
     url = args.config[0]
     utils = Utils(ConfStoreProvider(url))
     stop_event = Event()
-    local_dir = '/var/hare'
-    consul_starter = _start_consul(utils, stop_event, local_dir)
+    conf_dir = get_config_dir(url)
+    local_dir = f'{conf_dir}'
```

Comment:
why? what is the type of conf_dir? should be str, right?

Reply:
Done